.buildkite/pipeline.yml

```diff
@@ -13,8 +29,7 @@ steps:
   - label: ":pipeline: Build"
     key: "build"
     command: ".buildkite/commands/build-for-testing.sh"
-    env:
-      IMAGE_ID: xcode-12.5.1
+    env: *common_env
     <<: *use_plugins
```

Comment:
For consistency, we might want to update this, too, to use something like `plugins: *common_plugins`.

I want to hear what the reviewers have to say on my approach first, though.

Reply:
Definitely.

See also [here](https://github.com/wordpress-mobile/gutenberg-mobile/pull/3758#discussion_r678072609) and the full explanation/discussion (I went full verbose Olivier again …) [here](https://github.com/wordpress-mobile/gutenberg-mobile/pull/3764#discussion_r678088237) on @oguzkocer 's PR where we wonderw the same thing